src/atopile/front_end.py

```diff
@@ -452,7 +324,7 @@ def get_frozen(self) -> tuple:
 }
 
 
-class Bob(BasicsMixin, PhysicalValuesMixin, SequenceMixin, AtoParserVisitor):  # type: ignore  # Overriding base class makes sense here
+class Bob(BasicsMixin, SequenceMixin, AtoParserVisitor):  # type: ignore  # Overriding base class makes sense here
```

Comment:
Squashed the physical values mixin in here so it had access to the traceback stack